### ISSUE TITLE
[1.12.2] Browsing the Nasa Workbench crashes the game, again :D

### ISSUE THREAD

**Issue by wamc2017** [MJRLegends/Planet-Progression#9]
### ---Issue Report---

**Have you checked Closed Issues (if applicable & PLEASE CHECK BEFORE POSTING):** yes

### Description of Issue
Browsing the Nasa Workbench crashes the game, looks similar to #2 ;)
Also tried a selfcompiled version with the latest commit, same thing there...

### Steps to Reproduce Issue
1. Place a Nasa Workbench and click next in a New world

**Crash log/Log (if applicable):**
https://pastebin.com/25sgZEkX

**Screenshots (if applicable):** 

**Version of Mod using (PLEASE DONT USE 'Latest' OR 'Newest'):** 1.12.2-0.1.3

**Where did this bug happen:** Single Player

### Additional Information
Modlist: https://pastebin.com/rpxkqw2z

**Comment by MJRLegends** [comment]
Yeah i will fix this now and please never use self complied things

**Comment by MJRLegends** [comment]
Yeah i will fix this now and please never use self complied things
